### PR TITLE
chore: remove adpr env

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- if you want to switch env: => ➡️  change this `run: yarn build-prod` in pr.yaml
- if you want to use `dev env` and some `API prod env`: ➡️ `run: yarn build-dev` and update `API in .env.dev`
- of course, don't forget to revert all of them before merge

### CODE REVIEW

File: src/utils/keyframes.ts
Comment (namgold) [inline src/utils/keyframes.ts:1]: let's keep this

Comment (nguyenhoaidanh) [reply src/utils/keyframes.ts:1]: unused

Comment (namgold) [reply src/utils/keyframes.ts:1]: I will use it soon

Comment (nguyenhoaidanh) [reply src/utils/keyframes.ts:1]: reverted

**Review by namgold — APPROVED** [review]

**Pull request merged**